test/functional/connection/connection.ts

```diff
@@ -359,5 +359,33 @@ describe("Connection", () => {
         })));
 
     });
+
+    describe("Can change postgres default schema name", () => {
+        let connections: Connection[];
+        beforeEach(async () => {
+            connections = await createTestingConnections({ 
+                enabledDrivers: ["postgres"],
+                entities: [Post]
+            });
+        });
+        afterEach(() => closeTestingConnections(connections));        
+        it("schema name can be set", () => {
+            return Promise.all(connections.map(async connection => {
+                await connection.syncSchema(true);
+
+                const post = new Post();
+                post.title = "ChangeSchemaName";
+
+                const PostRepo = connection.getRepository(Post);
+                await PostRepo.persist(post);
+
+                const query = await connection.driver.createQueryRunner();
+                const schemaName = connection.driver.options.schemaName || "public";
```

Comment:
I think
```
> "test-schema" || "public"
"test-schema"
> undefined || "public"
"public"
```

and

console print `console.log(schemaName, connection.driver.options);` is
```
test-schema { type: 'postgres',
  host: 'localhost',
  port: 5432,
  username: 'test',
  password: 'test',
  database: 'test',
  schemaName: 'test-schema' }
public { type: 'postgres',
  host: 'localhost',
  port: 5432,
  username: 'test',
  password: 'test',
  database: 'test' }
```